app/lib/statement_pdf_factory.rb

```diff
@@ -4,8 +4,8 @@ class StatementPdfFactory
 
   @@statement_pdf_class = Settings.statement_pdf.class_name.constantize
 
-  def self.instance(*args)
-    @@statement_pdf_class.new(*args)
+  def self.instance(args, **kwargs)
```

Comment:
Did you looks at where/how this is invoked?  I'd guess the intent is to pass in attributes (a hash).

Reply:
@giladshanan the cases I've found look like `StatementPdfFactory.instance(@statement, download: true)` and `StatementPdfFactory.instance(@statement)`, so a keyword argument is being used sometimes.